locales/jp/translation.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Change locale folder name from `jp` to `jo` ([ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))